modules/virt-how-fs-overhead-affects-space-vm-disks.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-reserving-pvc-space-fs-overhead.adoc
+
+[id="virt-how-fs-overhead-affects-space-vm-disks_{context}"]
+= How file system overhead affects space for VM disks
+
+When you add a virtual machine disk to a persistent volume claim (PVC) that uses the `Filesystem` volume mode, you must ensure that there is enough space on the PVC for:
+
+* The virtual machine disk.
+* The space that the Containerized Data Importer (CDI) reserves for file system overhead, such as metadata.
+
+By default, CDI reserves 5.5% of the PVC space for overhead, reducing the space available for virtual machine disks by that amount.
```

Comment:
s/CDI/the CDI

Reply:
Oooh, this feels like "The Facebook," but I'll take it.